scripts/environment.sh

```diff
@@ -85,6 +85,10 @@ export PROVISIONING_URL_HOST="172.22.0.1"
 export CLUSTER_PROVISIONING_IP="172.22.0.2"
 export CLUSTER_URL_HOST="$CLUSTER_PROVISIONING_IP"
 
+export BM_RPOVISIONER_URL_HOST="172.22.0.1"
+export CLUSTER_BM_PROVISIONER_IP="172.22.0.2"
```

Comment:
this two names are confusing I think both means ironic ip ? or there is a difference between `BM_PROVISIONER` and `CLUSTER_BM_PROVISIONER`

Reply:
The situation is the same as with the original variables:
export PROVISIONING_URL_HOST="172.22.0.1"
export CLUSTER_PROVISIONING_IP="172.22.0.2"

Later in dev-env I can take look at these vars because indeed they seem to be redundant but the scope of the current PR is just to rename the variables.